pkg/api/v1/client/proxy.go

```diff
@@ -437,8 +437,9 @@ func (c ProxyScriptsAPI) GetExecutionArtifacts(executionID string) (artifacts te
 	return c.getArtifactsFromResponse(resp)
 
 }
+
 func (c ProxyScriptsAPI) DownloadFile(executionID, fileName, destination string) (artifact string, err error) {
-	uri := c.getURI("/executions/%s/artifacts/%s", executionID, fileName)
+	uri := c.getURI("/executions/%s/artifacts/%s", executionID, url.QueryEscape(fileName))
```

Comment:
I think it is better to use .Param() for the query params in the proxy.

Reply:
let's revisit this later